core/src/main/kotlin/com/alecstrong/sql/psi/core/psi/mixins/ForeignKeyClauseMixin.kt

```diff
@@ -18,3 +21,26 @@ internal abstract class ForeignKeyClauseMixin(
     return super.queryAvailable(child)
   }
 }
+
+fun SqlColumnDef.isForeignKey(): Boolean {
```

Comment:
After thinking again, this won't work in sqldelight with the mysql dialect supporting `ALTER TABLE DROP FOREIGN KEY` because this isn't a `SqlColumnDef`... I guess the only option is to put the keys into the schema too 🤔